From 873130e4268722a452718b63578234da7ca15147 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Wed, 14 Dec 2022 10:19:11 +0200
Subject: [PATCH] [OpenMP] Add a missing dllexport for the new function
 __kmpc_fork_call_if

This new function was added in b72f1ec9fbb14cd7d2f5112d2c52ef5cdd1aa94a,
but wasn't exported from the DLL on Windows.
---
 openmp/runtime/src/dllexports | 1 +
 1 file changed, 1 insertion(+)

diff --git a/openmp/runtime/src/dllexports b/openmp/runtime/src/dllexports
index 0699e7596841..ec3c6dcb5ea0 100644
--- a/openmp/runtime/src/dllexports
+++ b/openmp/runtime/src/dllexports
@@ -228,6 +228,7 @@
     __kmpc_for_static_init_4                136
     __kmpc_for_static_init_8                137
     __kmpc_fork_call                        138
+    __kmpc_fork_call_if                     294
     __kmpc_global_num_threads               140
     __kmpc_global_thread_num                141
     __kmpc_in_parallel                      142
-- 
2.34.1

